libs/stocks/feature-shell/src/lib/stocks/stocks.component.html

```diff
@@ -5,6 +5,7 @@
       placeholder="Symbol e.g AAPL"
       formControlName="symbol"
       value=""
+      (blur)="fetchQuote()"
```

Comment:
Utilizing the existing fetchQuote() function to leverage the form validation before dispatching the action.

Set to onBlur so that API call is made when the user is completed typing